.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: node_js
+node_js:
+ - '10.1.0'
+sudo: disabled
+
+cache: 
+  directories:
+    - "node_modules"
+
+
+install:
+  - npm install
+
+before_script:
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./node_modules/.bin/precise-commits --whitelist="./**/!(*.json)" --check-only --head=$TRAVIS_PULL_REQUEST_SHA --base=$(git merge-base HEAD $TRAVIS_BRANCH); fi'
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_PAGES  # Set in the settings page of your repository, as a secure variable
+  keep-history: true
```

Comment:
I think we can set this to `false` and just always push over top of what's there, which is nice if you make a mistake.

Reply:
We will need a separate gh-pages branch as a target, if we to drop the history. I don't want force pushed to screw up master.

Reply:
I manually created `gh-pages` so it will exist when needed.  OK, let's keep it how you have it for now, and tweak later if we need to.